Dockerfile

```diff
@@ -2,16 +2,60 @@ FROM ubuntu:16.04
 MAINTAINER Sharath Ganesh Pai <sharath@cliqz.com>
 ENV DEBIAN_FRONTEND noninteractive
 
+#Install the required packages. 1st Set is for FF and the 2nd for Ruby and FastLane.
 RUN dpkg --add-architecture i386 && \
     apt-get update && \
     apt-get upgrade -y && \
-    apt-get install -y autoconf2.13 build-essential ccache curl git lib32z1 \
-        libc6:i386 libncurses5:i386 libstdc++6:i386 mercurial openjdk-8-jdk \
-        python-dev python-pip python-setuptools unzip uuid \
-        wget xz-utils zip && \
+    apt-get install -y \
+        autoconf2.13 \
+        build-essential \
+        ccache \
+        curl \
+        git \
+        lib32z1 \
+        libc6:i386 \
+        libncurses5:i386 \
+        libstdc++6:i386 \
+        mercurial \
+        openjdk-8-jdk \
+        python-dev \
+        python-pip \
+        python-setuptools \
+        unzip \
+        uuid \
+        wget \
+        xz-utils \
+        zip && \
+    apt-get install -y \
```

Comment:
those two can be merged :-)

Reply:
I actually wanted to keep it separate to make it easier to track. The 1st set are the dependencies for FF and the next is our own. Mixing them both would make it slightly difficult to track later on.